sources/messages/configure_protocol.move

```diff
@@ -0,0 +1,62 @@
+#[allow(unused_field,unused_use,unused_const,unused_mut_parameter,unused_variable,unused_assignment)]
+module balanced::configure_protocol {
+    use std::string::{Self, String};
+    use std::option::{some, none};
+    use sui_rlp::encoder;
+    use sui_rlp::decoder;
+
+    public struct ConfigureProtocol has drop{
+        sources: vector<String>, 
+        destinations: vector<String>
+    }
+
+    public fun encode(req:&ConfigureProtocol, method: vector<u8>):vector<u8>{
+        let mut list=vector::empty<vector<u8>>();
+        vector::push_back(&mut list, encoder::encode(&method));
+
+        let source_encoded = encoder::encode_strings(&req.sources);
+        let dest_encoded = encoder::encode_strings(&req.destinations);
+
+        vector::push_back(&mut list, encoder::encode(&source_encoded));
+        vector::push_back(&mut list, encoder::encode(&dest_encoded));
+
```

Comment:
vector::push_back(&mut list,encoder::encode_strings(&req.sources));
          vector::push_back(&mut list,encoder::encode_strings(&req.destinations)); this should be enough . we dont need to encode the bytes from encode_strings.